contracts/lib.rs

```diff
@@ -258,6 +276,8 @@ pub mod prosopo {
         contract: AccountId,
         owner: AccountId,
         value: Balance,
+        payee: DappPayee,
```

Comment:
what are these for?

Reply:
This is tested in `test_get_random_active_provider_dapp_any`